src/mmw/js/src/projects/views.js

```diff
@@ -0,0 +1,160 @@
+"use strict";
+
+var Marionette = require('../../shim/backbone.marionette'),
+    App = require('../app'),
+    router = require('../router').router,
+    models = require('../modeling/models'),
+    modalModels = require('../core/modals/models'),
+    modalViews = require('../core/modals/views'),
+    containerTmpl = require('./templates/container.html'),
+    rowTmpl = require('./templates/projectRow.html');
+
+var ProjectsView = Marionette.LayoutView.extend({
+    template: containerTmpl,
+
+    ui: {
+        newProject: '.new-project'
+    },
+
+    events: {
+        'click @ui.newProject': 'newProject'
+    },
+
+    regions: {
+        rowContainer: '.project-rows-container'
+    },
+
+    onShow: function() {
+        var self = this,
+            projects = new models.ProjectCollection();
+        projects
+            .fetch()
+            .done(function() {
+                self.rowContainer.show(new ProjectRowsView({
+                    collection: projects
+                }));
+            });
+    },
+
+    newProject: function() {
+        App.currentProject = null;
```

Comment:
I could be wrong, but I seem to recall this happening in a few places now. We might want to make this a method of the App.